### PR TITLE
allow timeout to be configurable

### PR DESCRIPTION
**Pull request by achen2401**
https://www.pivotaltracker.com/story/show/180827785
Allow Inactivity timeout to be disabled by configuration (`ENABLE_INACTIVITY_TIMEOUT`).
If enabled, function as-is (`default; ENABLE_INACTIVITY_TIMEOUT=true`).
If disabled (`ENABLE_INACTIVITY_TIMEOUT=false`),  session will automatically be refreshed by a browser refresh with a popup modal indicating as such (see screenshot)
patientSearchTimeout

### CODE REVIEW

**Comment by achen2401** [comment]
@ivan-c Will you try these changes out and see if they are working as you think they should be, when you have a minute?

File: patientsearch/src/js/components/TimeoutModal.js [hunk 1]
```diff
@@ -158,23 +176,29 @@ export default function TimeoutModal() {
           <span className="error">Your current session has expired.</span>
         )}
         {expiresIn && expiresIn != 0 && (
-          <span>
-            Your session will expired in approximately{" "}
-            {getExpiresInDisplay(expiresIn)}.
-          </span>
+          <React.Fragment>
+            {!disabled && <span>
+              Your session will expired in approximately
```
Comment (ivan-c) [inline patientsearch/src/js/components/TimeoutModal.js:181]: I think it's out of scope for this PR, but I did notice when I put my laptop to sleep (running Firefox 91.5.0) and later woke it up, all the javascript seemed to pause on the page. The REST calls to /validate-token stopped, and I was never redirected away from the patient list.

I'll try reproducing again and let you know if I find more.

**Review by ivan-c — APPROVED** [review]
Looks great, thank you!!

I tested overnight and my session was successfully refreshed, and later automatically logged out when the Keycloak SSO Session Max Lifetime (10 hours) was reached

File: patientsearch/src/js/components/TimeoutModal.js (hunk 1 above)
Comment (achen2401) [reply patientsearch/src/js/components/TimeoutModal.js:181]: @ivan-c I was able to reproduce this.  I see this in the browser console, is this what you see?
Screen Shot 2022-01-25 at 12 02 11 PM

Comment (ivan-c) [reply patientsearch/src/js/components/TimeoutModal.js:181]: > @ivan-c I was able to reproduce this. I see this in the browser console, is this what you see? Screen Shot 2022-01-25 at 12 02 11 PM

@achen2401 thanks for trying! ...but that's not what I'm seeing when I sleep/unsleep my laptop with Firefox. When it happens on my laptop, I don't see **any** console output (or REST requests)/

Comment (achen2401) [reply patientsearch/src/js/components/TimeoutModal.js:187]: @ivan-c I **think** I addressed the issue.  Will you give it a try again when you have a moment?

Comment (ivan-c) [reply patientsearch/src/js/components/TimeoutModal.js:196]: Just to close the loop- tested and works great out of sleep

**Review by pbugni — APPROVED** [review]
Looks great @achen2401 - please note the lint error.

**Pull request merged**